src/metadata-storage.js

```diff
@@ -193,7 +193,9 @@ class MetadataStorage {
         utils.log.info(`window.sessionStorage unavailable. Reason: "${e}"`);
       }
     }
-    if (!str) false;
+    if (!str) {
```

Comment:
small nit: this is 
```
if (condition) { return false } else {return true} 
```

can be condensed slightly by skipping on the `if` and just returning `!!str`